CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Breaking
 
 ### Feature
+- Showing Version in History View @iFlameing
```

Comment:
@iFlameing "Show version number in history view" (changelog entries should be: lower case, present tense)